### PR TITLE
[FEATURE] Create siblidng-fade.md

### PR DESCRIPTION
**Pull request by thala321**
Add new css snippet. Fade out siblings around a hovered item.

### CODE REVIEW

**Comment by equinusocio** [comment]
This this the effect of this snippet? 

https://jsfiddle.net/po6wvcsu/2/

If yes, you written a lot of code :)

**Comment by thala321** [comment]
Oh yes, my code was really an overkill.

Vào 02-03-2018 2:02 CH, "Mattia Astorino" <notifications@github.com> đã
viết:

This this the effect of this snippet?

https://jsfiddle.net/po6wvcsu/2/

If yes, you written a lot of code :)

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/atomiks/30-seconds-of-css/pull/49#issuecomment-369840211>,
or mute the thread

**Comment by equinusocio** [comment]
So, can you update the snippet?

File: siblidng-fade.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+### Sibling fade
```
Comment (Chalarangelo) [inline siblidng-fade.md]: The file name is wrong, please rename to `sibling-fade`.

File: siblidng-fade.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+#### Explanation
+
+1. `.sibling-fade { visibility: hidden; }` Prevents :hover from triggering in the gaps between items.
+2. `.sibling-fade > * { visibility: visible; }` Brings the child items back in, even though the parent is hidden.
+3. `.sibling-fade > * { transition: opacity 150ms linear 100ms, transform 150ms ease-in-out 100ms; }` Makes the fades smooth with a slight delay to prevent jumps as the mouse moves between items.
+4. `.sibling-fade:hover > * { opacity: 0.4; transform: scale(0.9); }` Fade out all items when the parent is hovered.
+5. `.sibling-fade > *:hover { opacity: 1; transform: scale(1); transition-delay: 0ms, 0ms; }` Fade in the currently hovered item.
+
+#### Browser support
+
```
Comment (Chalarangelo) [inline siblidng-fade.md]: If there are no caveats, you should add the appropriate comment about this here, like `<span class="snippet__support-note">✅ No caveats.</span>`.

File: siblidng-fade.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+#### Made by: @shshaw
```
Comment (Chalarangelo) [inline siblidng-fade.md]: Author credit is not added into the snippets, but rather shown via the Github interface on the repository. Please remove this.

**Pull request closed without merging**